### PR TITLE
Ascii product character

### LINKED ISSUE #10413: ascii pprint of ProductSet uses non-ascii multiplication symbol
````
>>> srepr(A)
'ProductSet(FiniteSet(Rational(13, 111), Integer(2), Integer(3)), FiniteSet(exp(Integer(10)), Mul(Rational(1, 2), pi)))'

>>> pprint(A, use_unicode=True)
⎧ 13      ⎫   ⎧π   10⎫
⎨───, 2, 3⎬ × ⎨─, ℯ  ⎬
⎩111      ⎭   ⎩2     ⎭

>>> pprint(A, use_unicode=False)
  13           pi   10 
{---, 2, 3} × {--, e  }
 111           2       
````

Last one should be `x` not `\xd7`.  See `printing/pretty/pretty.py`.

Possible fix might be:
`prod_char = U('MULTIPLICATION SIGN') if self._use_unicode else 'x'`

Also needs a test---can be simplier than mine.

### PR DESCRIPTION
**Pull request by tokencolour**
Fixes #10413

### CODE REVIEW

**Comment by cbm755** [comment]
wow!  fast!   ;-)

1.  I think we can do better than the `\xd7` escaped string.
2.  it will also need a new test (for pprint with `use_unicode=false`).

**Comment by tokencolour** [comment]
@cbm755 , Thank you for your comments. I'll add a simpler test. What can we do so that it is better than `\xd7`?

**Comment by tokencolour** [comment]
@cbm755 , I have added the tests. Do tell if you suggest improvements. Thank you.

**Comment by tokencolour** [comment]
By doing better, do you mean using a common identifier for unicode and ascii multiplication symbol, perhaps?

**Comment by cbm755** [comment]
My original bug report has what I think it should be.  Or maybe its `u("\N{MULT...}")`.

File: sympy/printing/pretty/tests/test_pretty.py [hunk 1]
```diff
@@ -3147,6 +3147,14 @@ def test_ProductSet_paranthesis():
     a, b, c = Interval(2, 3), Interval(4, 7), Interval(1, 9)
     assert upretty(Union(a*b, b*FiniteSet(1, 2))) == ucode_str
 
+def test_ProductSet_prod_char():
+    from sympy import Interval
```
Comment (cbm755) [inline sympy/printing/pretty/tests/test_pretty.py]: import needed here?

File: sympy/printing/pretty/tests/test_pretty.py [hunk 2]
```diff
@@ -3147,6 +3147,14 @@ def test_ProductSet_paranthesis():
     a, b, c = Interval(2, 3), Interval(4, 7), Interval(1, 9)
     assert upretty(Union(a*b, b*FiniteSet(1, 2))) == ucode_str
 
+def test_ProductSet_prod_char():
```
Comment (cbm755) [inline sympy/printing/pretty/tests/test_pretty.py]: `test_ProductSet_prod_char_issue_10413`?

**Comment by tokencolour** [comment]
@cbm755 , I have made the changes that you suggested. Thank you for your comments. Please take a look at this.

**Comment by cbm755** [comment]
+1, will merge in 24 hours

**Comment by tokencolour** [comment]
Thank you, @cbm755 !

**Pull request merged**